torch/_dynamo/guards.py

```diff
@@ -480,9 +480,14 @@ def NN_MODULE(self, guard: Guard):
             )
             return
         name = self.check_fn_manager.add_extra_closure_var("__nn_module_guard", g)
-        # debug_msg is only for debugging help and goes to kwargs of guard call,
-        # which is ignored.
-        self._produce_guard_code(guard, [f'{name}({ref}, debug_msg="{g}")'])
+        if guards_log.isEnabledFor(logging.DEBUG):
+            # Avoid debug_msg related python bytecode overhead in the runtime.
+
+            # debug_msg is only for debugging help and goes to kwargs of guard call,
+            # which is ignored.
+            self._produce_guard_code(guard, [f'{name}({ref}, debug_msg="{g}")'])
```

Comment:
Can we make this more symmetric with how regular guard debugging? Specifically, pass the debug message to produce_guard_code, and propagate it to the debugging logic later

Reply:
Will look more into this. Let me get this one in to avoid perf regressions if any.